pandas/core/groupby.py

```diff
@@ -3187,9 +3187,9 @@ def _iterate_slices(self):
                 continue
             yield val, slicer(val)
 
-    def _cython_agg_general(self, how, alt=None, numeric_only=True):
+    def _cython_agg_general(self, how, alt=None, numeric_only=True, skipna=True):
```

Comment:
make this all consistent either ``skipna=True`` or ``skipna=None`` (this is different than below)